samples/gamma-fade/Makefile

```diff
@@ -1,5 +1,5 @@
-XBE_TITLE = gamma-fade
-GEN_XISO = $(XBE_TITLE).iso
+XBE_TITLE = "nxdk sample - gamma-fade"
```

Comment:
While the quotes are necessary when passing the parameter through the shell to extract-xiso, this is not a good place to add them. People might want to use the `XBE_TITLE` variable for additional stuff, and not realize that the quotation marks are actually part of the string here (they're not in shell scripts, but they are in Makefiles).

Use escaping instead: `nxdk\ sample\ -\ gamma-fade`.